src/main/scala/org/broadinstitute/dsde/rawls/dataaccess/slick/DataAccess.scala

```diff
@@ -48,4 +48,36 @@ trait DataAccess
       workflowFailureQuery.schema ++
       workflowMessageQuery.schema ++
       workflowAuditStatusQuery.schema
+
+  def truncateAll: WriteAction[Int] = {
+    // important to keep the right order for referential integrity !
```

Comment:
This is a good idea.

Pre-PR behavior is:
1. Create DB tables at beginning of each test
2. Drop DB tables at end of each test
Breaking mid-test: DB exists for next iteration of step 1, causing failures 

Current PR behavior is:
1. Create DB tables at beginning of each test suite (once per Spec file)
2. Truncate DB tables at end of each test
3. Drop DB tables at end of each test suite
Breaking mid-test: DB exists for next iteration of step 1, causing failures 

Here's a new plan:
1. Before the first test runs, detect whether the DB exists, and if so: drop all tables
2. Before the first test runs, create DBtables
3. Truncate DB tables at end of each test
Breaking mid-test: DB exists for next iteration of step 1, but that's OK